02-peer-protocol.md

```diff
@@ -968,6 +997,42 @@ additions.
 
 The accepting node:
     - MAY respond with a `funding_satoshis` value of zero.
+    - if the `option_will_fund` tlv was sent in `open_channel2`:
+        - if they decline to contribute funds or to be paid for their
+          contributions:
+            - SHOULD omit the `will_fund` tlv
+        - if they decide to accept the offer:
+            - MUST include a `will_fund` tlv
+            - MUST set `funding_satoshis` to a value greater than 0msat
+            - MAY provide a `funding_satoshis` of any amount.
+            - MUST set `funding_fee_base_sat` to the base fee
+              (in satoshi) it will charge for the `funding_satoshis`
+            - MUST set `funding_fee_proportional_basis` to the amount
+              (in thousandths of satoshi) it will charge per `funding_satoshi`
+            - MUST set `funding_weight` to the weight they
+              will contribute to this channel, to fund the request.
```

Comment:
Not *quite*, this is the weight that the initiating node will pay for (I suspect this language predates that change)

Reply:
Good point, will update.